chat-plugins/calculator.js

```diff
@@ -0,0 +1,115 @@
+'use strict';
+
+function isNumeric(str) {
+	return !isNaN(parseFloat(str)) && isFinite(str);
```

Comment:
You shouldn't pass `str` directly to `isFinite`; it's the same idea as `isNaN`.

Reply:
Ah, didn't realize it was redundant. Fixed,